examples/http-server-using-redis/go.mod

```diff
@@ -0,0 +1,7 @@
+module example.com/redis
```

Comment:
Why are we adding modules for the examples.

Reply:
Removed the same